cachi2/core/models/property_semantics.py

```diff
@@ -87,6 +92,8 @@ def to_properties(self) -> list[Property]:
             props.append(Property(name="cachi2:pip:package:binary", value="true"))
         if self.bundler_package_binary:
             props.append(Property(name="cachi2:bundler:package:binary", value="true"))
+        if self.add_rpm_summary:
+            props.append(Property(name="cachi2:bundler:package:binary", value="true"))
```

Comment:
Won't this property be a bool in the output after the component properties are merged? Also, shouldn't the name of this Property be `cachi2:rpm_summary`?